charts/apps/kubeflow-pipelines/rds-s3-static/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml

```diff
@@ -1,4 +1,4 @@
-2023/03/05 23:26:52 well-defined vars that were never replaced: kfp-app-name,kfp-app-version
+2023/03/17 11:45:04 well-defined vars that were never replaced: kfp-app-name,kfp-app-version
```

Comment:
bug

Reply:
this was not a bug by helmify. kustomize v5.0.0 automatically generate these warning message at the header of the yaml file